### PR TITLE
Move nasa-veda hub URL

### PR DESCRIPTION
**Pull request by yuvipanda**
We keep redirects in place from the old URL, in case there are external links already present.

The OAuth callback URL needs to be changed at
https://github.com/organizations/2i2c-org/settings/applications/2096065 just before merging.

Ref https://github.com/2i2c-org/infrastructure/issues/3029

### CODE REVIEW

**Review by consideRatio — APPROVED** [review]

**Comment by yuvipanda** [comment]
Let's wait for explicit approval in https://github.com/2i2c-org/infrastructure/issues/3029#issuecomment-1693843216 before going through with this.

**Comment by yuvipanda** [comment]
@freitagb is going to use this as an opportunity to use a brand new domain. So let's hold off for a bit more.

File: config/clusters/nasa-veda/support.values.yaml
Comment (consideRatio) [inline config/clusters/nasa-veda/support.values.yaml:32]:
```suggestion
      to: staging.veda.2i2c.cloud
    - from: grafana.nasa-veda.2i2c.cloud
      to: grafana.veda.2i2c.cloud
```

Let's get grafana updated as well (and the associated callback URL for the oauth2 application)

**Comment by yuvipanda** [comment]
Ping @freitagb - what do we do with this one?

**Comment by consideRatio** [comment]
Ping @freitagb - should we move forward on this?

**Comment by yuvipanda** [comment]
I've re-pinged @freitagb on slack now.

**Comment by freitagb** [comment]
yes, can we move forward with migrating to openveda.cloud?

**Comment by freitagb** [comment]
@yuvipanda can we use hub.openveda.cloud

**Comment by slesaad** [comment]
@yuvipanda done!

**Comment by yuvipanda** [comment]
@slesaad hmm, I'm not sure. I guess I'll just wait a little more and see?

**Comment by yuvipanda** [comment]
@slesaad they still don't resolve for me - are they resolving for you?

**Comment by slesaad** [comment]
@yuvipanda they are not 🤔 i wonder why. let me delete and recreate them and see if that helps. I don't know what else to try 😅

**Comment by yuvipanda** [comment]
@slesaad yeah i don't see it still. I'm not sure what the problem is. I still see it return an `NXDOMAIN`, which means the authoritative DNS server doesn't think it exists still. Maybe there's an extra 'rollout' step or something? I'm not sure.

I also see that `openveda.cloud` itself doesn't really resolve to anything, so maybe that's also part of the issue? Whoever is managing that entire domain may be able to help.

**Comment by yuvipanda** [comment]
Thanks @slesaad!

**Pull request merged**
